catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -449,4 +453,29 @@ public DeleteResponse process(DeleteResponse deleteResponse) throws PluginExecut
         modified, effective, expiration, created
     }
 
+    public void pingSubscriptionResponseHandlers() {
+
+        //// TODO (RWY) - make pool size configurable possibly move pool to constructor and make final.
+        ForkJoinPool forkJoinPool = new ForkJoinPool(4);
```

Comment:
If you were using the global pool, I could understand this as a shortcut, but you're not using fork/join semantics. Any reason you're not just creating an executor service at the class level?